platform/commonUI/general/src/controllers/ObjectInspectorController.js

```diff
@@ -108,8 +110,13 @@ define(
 
                 getMetadata();
             });
-        }
 
+            this.InspectorMutation.listen(function () {
+                if (!angular.equals($scope.metadata, $scope.ngModel.selectedObject.useCapability('metadata'))) {
```

Comment:
Is the `angular.equals` necessary? Generally mutations are rare and user-initiated, so I'd expect performance to be fine.

Along with changes from above, this could reduce the changeset to something like:

```
var mutation = $scope.ngModel.selectedObject.getCapability('mutation');
var unlisten = mutation.listen(getMetadata);
$scope.$on("$destroy", unlisten);
```